### PR TITLE
fix dbfs file format

### PR DESCRIPTION
**Pull request by SammyEK**
Signed-off-by: Sammy El Khammal <sammy.elkhammal@kensu.io>

### CODE REVIEW

File: kensu/pandas/data_frame.py [hunk 1]
```diff
@@ -757,7 +762,12 @@ def k_to_format(self, result_regex, *args, **kwargs):
         if location is None and len(args) > 0:
             def get_absolute_path(path):
                 import os
-                return 'file:'+str(os.path.abspath(path))
+                for prefix in ["abfs", "/abfs", "dbfs", "/dbfs"]:
```
Comment (vidma) [inline kensu/pandas/data_frame.py:765]: pasted

File: kensu/pandas/data_frame.py [hunk 2]
```diff
@@ -450,9 +450,14 @@ def k_to_format(self, result_regex, *args, **kwargs):
         if location is None and location != 'BigQuery Table' and len(args) > 0:
             def get_absolute_path(path):
                 import os
-                return 'file:'+str(os.path.abspath(path))
+                for prefix in ["abfs", "/abfs", "dbfs", "/dbfs"]:
```
Comment (vidma) [inline kensu/pandas/data_frame.py:453]: pasted

File: kensu/numpy/__init__.py [hunk 3]
```diff
@@ -102,9 +102,15 @@ def wrapper(*args, **kwargs):
 
         loc = args[0]
         df = args[1]
+
         def get_absolute_path(path):
             import os
-            return 'file:'+str(os.path.abspath(path))
+            for prefix in ["abfs", "/abfs", "dbfs", "/dbfs"]:
+                if path.startswith(prefix):
```
Comment (vidma) [inline kensu/numpy/__init__.py:109]: pasted

File: kensu/pandas/data_frame.py [hunk 4]
```diff
@@ -833,6 +843,12 @@ def wrapper(*args, **kwargs):
 
         def get_absolute_path(path):
             import os
+            for prefix in ["abfs","/abfs","dbfs","/dbfs"]:
```
Comment (vidma) [inline kensu/pandas/data_frame.py:846]: pasted

**Review by vidma — CHANGES_REQUESTED** [review]
this should be extracted as a generic shared function in `helpers` or `utils` package (or similar) and reused from multiple places

**Review by vidma — CHANGES_REQUESTED** [review]
hmm... and most importantly... isn't dbfs different than file? It's like HDFS, and shared among the multiple nodes/computers, so in my opinion it should be distinguished!

> Databricks File System (DBFS) is a distributed file system mounted into a Databricks workspace and available on Databricks clusters. 
https://docs.databricks.com/data/databricks-file-system.html

File: kensu/pandas/data_frame.py (hunk 4 above)
Comment (SammyEK) [reply kensu/pandas/data_frame.py]: done

**Comment by SammyEK** [comment]
> hmm... and most importantly... isn't dbfs different than file? It's like HDFS, and shared among the multiple nodes/computers, so in my opinion it should be distinguished!
> 
> > Databricks File System (DBFS) is a distributed file system mounted into a Databricks workspace and available on Databricks clusters.
> > https://docs.databricks.com/data/databricks-file-system.html

I hereby use the same naming for the file location as in the spark client

**Review by andypetrella — APPROVED** [review]

**Comment by vidma** [comment]
> I hereby use the same naming for the file location as in the spark client

interesting. we should check that if that's the case...

**Pull request merged**